### DISCUSSION TITLE
latent heat of snow fusion

### DISCUSSION THREAD

**Discussion by dsidoren** [FESOM/fesom2#263] [Ideas]
https://github.com/FESOM/fesom2/blob/7730f1cde958e086bc30bfd0e46514cd16dde908/src/ice_thermo_oce.F90#L296

@JanStreffing @qiangclimate in the standalone FESOM the latent heat of snow fusion is there but commented out. Qiang knows more detail but we can copy this line into ice_thermo_cpl.F90 as well so you do not need any extra manipulations on the OpenIFS side.

**Comment by qiangclimate** [comment]
241  Qatmocn = -a2ohf
-->
        Qatmocn = -a2ohf + snow * 1000.0 / rhoice*cl   !here snow is in unit of m of pure water

**Comment by JanStreffing** [comment]
> https://github.com/FESOM/fesom2/blob/7730f1cde958e086bc30bfd0e46514cd16dde908/src/ice_thermo_oce.F90#L296
> 
> @JanStreffing @qiangclimate in the standalone FESOM the latent heat of snow fusion is there but commented out. Qiang knows more detail but we can copy this line into ice_thermo_cpl.F90 as well so you do not need any extra manipulations on the OpenIFS side.

Those modifications on OpenIFS side are already there. We need to be careful putting this on the FESOM2 side as well. We don't want both active.

**Reply by dsidoren** [reply]
if you want to keep it on the OpenIFS side is fine (will do it with the directive). We need to reactivate it for ocean only and AWICMs < 3 (upon someone agrees to test). Only Qiang knows why it is commented out but probably he forgot :)

**Comment by qiangclimate** [comment]
yes, I added the code ten years ago, but then commented it out later for reasons that I forgot :((
